ui/src/app/views/concept-homepage/component.html

```diff
@@ -0,0 +1,83 @@
+<app-top-box>
+  <clr-tabs>
+    <clr-tab>
+      <button clrTabLink (click)="searching = false;">Concepts</button>
+    </clr-tab>
+    <clr-tab>
+      <button clrTabLink>Concept Sets</button>
+    </clr-tab>
+  </clr-tabs>
+  <div style="display: flex; align-items: center; margin-top: 1.5%;">
+    <clr-icon shape="search" style="position: absolute; height: 1rem; width: 1rem; fill: #216FB4; left: 5.3%;"></clr-icon>
+    <input type="text" id="concept-search-input" [(ngModel)]="searchTerm" placeholder="Search concepts in domains">
+    <span role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right btn-search-container">
+      <button class="btn btn-secondary btn-search" (click)="searchButton()" [disabled]="getSearchDisabled">
+        Search
+        <span style="padding-left: 0.5rem; display: flex;" *ngIf="searchLoading"><span class="spinner spinner-sm"></span></span>
+      </button>
+      <span *ngIf="searchTermNotLongEnough" class="tooltip-content">Search must be at least three letters long</span>
+    </span>
+    <clr-checkbox class="standard-concepts-checkbox" [(clrChecked)]="standardConceptsOnly">
+      Standard concepts only
+    </clr-checkbox>
```

Comment:
I have a UX philosophical objection to using a checkbox as a search button which is what this effectively does (see bullet 2 in "When to Use Which Widget" here https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/). However, I wouldn't hold this PR up on that. I think we should consider revisiting this at some point.

Reply:
Talked offline, this is going to be a followup with Karthik and Lou. Currently not searching on select/deselect though because it adds complexity.

Reply:
👍